brain_games/scripts/brain_games.py

```diff
@@ -0,0 +1,2 @@
+def main():
+    print("Welcome to the Brain Games!")
```

Comment:
[optional] add `if __name__ == "__main__": ...` section if you want to be able to run this script without `poetry run`